### DISCUSSION TITLE
kpanda/user-guide/storage/sc/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#1698] [Announcements]
# kpanda/user-guide/storage/sc/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/kpanda/user-guide/storage/sc/

**Comment by liyuerich** [comment]
能加一下授权命名空间怎么使用的吗？

**Comment by liyuerich** [comment]
快照/扩容：开启后，基于该存储池的数据卷/数据卷声明才能支持扩容和快照功能，但 前提是底层使用的存储驱动支持快照和扩容功能。

能说明一下怎么检查底层是否使用了这些功能，如果没有怎么添加吗？
